src/coreclr/vm/frames.cpp

```diff
@@ -929,24 +930,13 @@ GCFrame::GCFrame(Thread *pThread, OBJECTREF *pObjRefs, UINT numObjRefs, BOOL may
 
 #endif // USE_CHECKED_OBJECTREFS
 
+    m_Next          = NULL;
+    m_pCurThread    = NULL;
```

Comment:
This is unnecessary.

Reply:
It is now because I assert in the `Push()` the state of the members are `NULL`.

Reply:
If it is just to make asserts happy, it can be debug-only.